ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12-periodics.yaml

```diff
@@ -21694,105 +21694,6 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
-- agent: kubernetes
-  cluster: build04
-  cron: 8 16 27 * *
-  decorate: true
-  decoration_config:
-    skip_cloning: true
-  extra_refs:
-  - base_ref: release-4.12
-    org: openshift
-    repo: openshift-tests-private
-  labels:
-    ci-operator.openshift.io/cloud: azure4
-    ci-operator.openshift.io/cloud-cluster-profile: azure-qe
-    ci-operator.openshift.io/variant: amd64-nightly-openshift-logging-5.7
-    ci.openshift.io/generator: prowgen
-    job-release: "4.12"
-    pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-openshift-logging-5.7-azure-ipi-fips-f28
```

Comment:
@anpingli I think we can adjust the frequency. Besides, any changes to the jobs should be updated under `ci-operator/config/openshift/openshift-tests-private/` directory, and then run `make jobs` or `make update` to update this file.